docs/Getting Started/FreeBSD.rst

```diff
@@ -0,0 +1,129 @@
+FreeBSD
+=======
+
+|ZoF-logo|
+
+Installation on FreeBSD
+-----------------------
+
+OpenZFS is available in the FreeBSD ports tree as sysutils/openzfs and
+sysutils/openzfs-kmod. It can be installed on FreeBSD stable/12 or
+later.
+
+The ZFS utilities will be installed in /usr/local/sbin/, so make sure
+your PATH gets adjusted accordingly.
+
+To load the module at boot, put ``openzfs_load="YES"`` in
+/boot/loader.conf, and remove ``zfs_load="YES"`` if migrating a ZFS
+install.
+
+Beware that the FreeBSD boot loader does not allow booting from root
+pools with encryption active (even if it is not in use), so do not try
+encryption on a pool you boot from.
+
+Development on FreeBSD
+----------------------
+
+The following dependencies are required to build OpenZFS on FreeBSD:
+
+-  FreeBSD sources in /usr/src or elsewhere specified by SYSDIR in env.
+   If you don't have the sources installed you can install them with
+   svnlite.
+
+   Install source For FreeBSD 12:
+   ::
+
+     svnlite checkout https://svn.freebsd.org/base/stable/12 /usr/src
+
+   Install source for FreeBSD Current:
+   ::
+
+     svnlite checkout https://svn.freebsd.org/base/head/ /usr/src
+
+-  Packages for build:
+   ::
+
+      pkg install \
+          autoconf \
+          automake \
+          autotools \
+          bash \
```

Comment:
We no longer need bash for building ZFS.

Reply:
Moved bash to tests requirements then, IIRC there were some on bash.